src/codex_plus/main_sync_cffi.py

```
@@ -154,6 +246,399 @@ async def health():
+            forbidden_paths=agent_data.forbidden_paths,
+            capabilities=set(agent_data.capabilities),
+            version=agent_data.version,
+            author=agent_data.author,
+            tags=agent_data.tags
+        )
+
+        # Validate configuration
+        validation_issues = config.validate()
+        if validation_issues:
+            logger.warning(f"Validation issues for new agent: {validation_issues}")
+            raise HTTPException(status_code=400, detail=f"Validation errors: {', '.join(validation_issues)}")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)